src/MarketPlace.sol

```diff
@@ -35,9 +35,17 @@ contract MarketPlace {
         Notional // 8
     }
 
+    mapping(uint8 => address) public adapters;
+
+    // method sets adapters using an array of uint8's and array of addresses
+    function setAdapters(address[] calldata a) external {
```

Comment:
Should this method be `authorized`?

Reply:
Oops yup, I had already fixed this locally but hadnt pushed up!